### PR TITLE
Improve for 2022.1

### PR DESCRIPTION
**Pull request by keryell**
Mainly using a part for Vitis IP which is installed by default.

### CODE REVIEW

File: sycl/test/vitis/simple_tests/single_task_vector_add.cpp [hunk 1]
```diff
@@ -4,7 +4,7 @@
 // RUN: %clangxx -std=c++20 -fsycl -fsycl-targets=%sycl_triple %s -o %t.dir/exec.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.dir/exec.out
 // Test that we can get a fast reports by adding --vitis-ip-part
-// RUN: %clangxx -std=c++20 -fsycl -fsycl-targets=%sycl_triple %s -o %t.dir/exec2.out --vitis-ip-part=xc7vx330t-ffg1157-1
+// RUN: %clangxx -std=c++20 -fsycl -fsycl-targets=%sycl_triple %s -o %t.dir/exec2.out --vitis-ip-part=xcu200-fsgd2104-2-e
```
Comment (Ralender) [inline sycl/test/vitis/simple_tests/single_task_vector_add.cpp:7]: What we should do here is figure out inside lit what part id is available on the system and setup a substitution for it. similarly to what we do for "%opencv4_flags"
not everyone will have an u200

File: sycl/doc/GettingStartedXilinxFPGA.md [hunk 2]
```diff
@@ -915,7 +915,7 @@ treating compiler warnings as errors and producing a compiler database
 to be used by tools like LSP server like `clangd`:
 
 ```bash
-python3 $SYCL_HOME/llvm/buildbot/configure.py --cuda \
+python3 $SYCL_HOME/llvm/buildbot/configure.py --xrt --cuda \
```
Comment (Ralender) [inline sycl/doc/GettingStartedXilinxFPGA.md]:
```suggestion
python3 $SYCL_HOME/llvm/buildbot/configure.py --xrt --cuda --shared-libs \
```
shared libs greatly speed up the build so Ithink it should be added by default

**Review by Ralender — COMMENTED** [review]
Seems good, but I added some suggestion for improvement

File: sycl/test/vitis/simple_tests/single_task_vector_add.cpp (hunk 1 above)
Comment (keryell) [reply sycl/test/vitis/simple_tests/single_task_vector_add.cpp:7]: I think it is another problem.
Not everybody has an Alveo U200 platform installed but the `xcu200-fsgd2104-2-e` part comes de facto with Vivado.
But anyway, having the part should be configurable too.

File: sycl/doc/GettingStartedXilinxFPGA.md (hunk 2 above)
Comment (keryell) [reply sycl/doc/GettingStartedXilinxFPGA.md]: Good point! I thought it was the case by default.

File: sycl/test/vitis/simple_tests/single_task_vector_add.cpp (hunk 1 above)
Comment (Ralender) [reply sycl/test/vitis/simple_tests/single_task_vector_add.cpp:7]: if xcu200-fsgd2104-2-e comes by default with Vitis/Vivado its fine keeping it.

**Review by Ralender — APPROVED** [review]

**Pull request merged**